### ISSUE TITLE
ERR_REQUIRE_ESM .cjs etc

### ISSUE THREAD

**Issue by geeklystrips** [fcamarlinghi/expresso#26]
Attempting to build, getting multiple errors that seem to be due to current versions of node_modules dependencies.

- Error [ERR_REQUIRE_ESM]: Must use import to load ES Module
- require() of ES modules is not supported.
- Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from .../node_modules/cpy/package.json.

Tried downgrading to 2.6.6, still failing.
npm install package@2.6.6 --legacy-peer-deps

Any advice on making sure local setup matches the one used by the last user who committed?

Thanks

**Comment by fcamarlinghi** [comment]
NPM should automatically take care of matching the dependencies (via the provided _package-lock.json_ file). I just tried cloning and building locally (following the steps in the project readme) and it worked using Node 16.15 and NPM 8.14 on a Windows 10 machine.

Could you share more info about your setup?

**Comment by geeklystrips** [comment]
You were right!
I reverted everything and was able to get my setup to work, sort of (macOS Monterey 12.4)

My initial mistake was to npm audit fix because there were several warnings. Turns out it's best to leave warnings for now.
I realize was referring to "package" 2.6.6 in the above comment, I meant node-fetch. Looks like it was introduced by the audit fix.

<img width="633" alt="NodeFetch" src="https://user-images.githubusercontent.com/12120428/179157628-4eac864a-ea2d-4761-a1c4-42aa8b80d043.png">

Since I was still running into issues building on macOS, I switched to a PC setup (Win10, 19044.1826) and was able to get a functional ZXP that works on macOS (can't test on the PC machine for now)

I think there is an issue with the local settings file on macOS.
01:44:43.705 [CEP] [DEBUG] requestOpenExtension: com.expresso.exporter.settings undefined 

Closing this current issue. Thanks for your help!
